src/licensedcode/data/licenses/crapl-0.1.yml

```diff
@@ -0,0 +1,8 @@
+key: crapl-0.1
+short_name: CRAPL v0 BETA 1
+name: CRAPL v0 BETA 1
+category: Academic
```

Comment:
Thank for this and the other changes. I need @DennisClark input on the category before I can merge